### PR TITLE
Crude oil PR 

### PR DESCRIPTION
**Pull request by mkhraisha**
This PR adds Crude Oil:

Some issues with testing similar to #89 looking for a cleaner solution for it.

Will be issuing a second PR for Gas/ QP-INBOND soon.

### CODE REVIEW

**Comment by mkhraisha** [comment]
Theres new conflicts due to merge of cmtr, will be updating soon

**Review by mprorock — APPROVED** [review]
will need to pull latest changes and merge with your branch before we can merge the pr

**Comment by OR13** [comment]
^ all these files:

```
docs/contexts/traceability-v1.jsonld
docs/index.html
docs/intermediate.json
```

should be git ignored and built on PR merges... and we would have way less conflicts.

**Comment by mkhraisha** [comment]
will update gitignore in my PR then

**Comment by mprorock** [comment]
@mkhraisha while you are in there, do you mind running npm lint and cleaning up anything (line lengths, etc) that may be breaking ci?

**Comment by mkhraisha** [comment]
There appears to be a number of errors and there are changes  to the generate-credential-examples folder, so i'll be updating the oil samples to reflect them.

**Comment by mkhraisha** [comment]
Fixed lint issues and added the files to `gitignore` however they were still tracked so i removed them from cache and recommited. This should remove the files from your local repos on next pull.

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,7 @@
 node_modules
 .DS_Store
 .vscode
-.idea
\ No newline at end of file
+.idea
+docs/contexts/traceability-v1.jsonld
+docs/index.html
```
Comment (OR13) [inline .gitignore:6]:
```suggestion
```

Comment (OR13) [reply .gitignore:6]: we need to keep this until we add a github action.

**Comment by OR13** [comment]
Actually I am going to take this PR and do some much needed cleanup

**Pull request merged**